MAUI/Cartesian-Charts/StackedArea.md

```diff
@@ -101,4 +101,4 @@ this.Content = chart;
 
 {% endtabs %}
 
-![Stacking Area Chart in MAUI](Chart-types_images\StackedAreaChart.png)
```

Comment:
Change the folder format
<img width="625" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/61148953/82a5a840-dbf8-41c6-8043-fa0e97e65622">

Reply:
Changed